CRM/Core/BAO/Website.php

```diff
@@ -84,7 +84,7 @@ public static function process($params, $contactID, $skipDelete) {
         self::create($values);
       }
       elseif ($skipDelete && !empty($values['id'])) {
-        self::del($values['id']);
+        static::deleteRecord($values['id']);
```

Comment:
`deleteRecord()` takes an array
```suggestion
        static::deleteRecord($values);
```